index.js

```diff
@@ -139,21 +142,35 @@ function config () {
         map: argv.map
       }
     }
-  } else {
-    return postcssrc()
-    .catch(err => {
+  }
+
+  return postcssrc()
+    .catch((err) => {
       if (err.message.indexOf('No PostCSS Config found') === -1) throw err
-      else return {plugins: [], options: {}}
+      return { plugins: [], options: {} }
     })
-  }
 }
 
-function errorHandler (err) {
+function error (err) {
   try {
     spinner.fail()
   } catch (e) {
     // Don't worry about this
   }
-  console.error(err)
+
+  // PostCSS Error
+  if (err.name === 'CssSyntaxError') {
+    err.message = err.message
+      .replace(`${err.file}:`, '[')
+      .replace(/:\s/, '] ')
+
+    console.log(chalk.bold.red('\n', err.message))
+    console.log('\n', err.showSourceCode(), '\n')
+
+    return
+  }
+  // PostCSS Config Error
+  console.log(chalk.bold.red(err.message))
```

Comment:
I have a question about process.exit(1). Is it mandatory, bc it  throws ugly npm error + npm-debug.log :), is throw not enough to terminate?